MapleServer2/Data/Static/MapMetadataStorage.cs

```diff
@@ -138,4 +138,26 @@ public static int GetDistanceToNextBlockBelow(int mapId, CoordS coord, out MapBl
         block = null;
         return 99;
     }
+
+    public static bool MapIsInstancedOnly(int mapId)
+    {
+        MapMetadata mapMetadata = GetMetadata(mapId);
+        if (mapMetadata is null)
+        {
+            return false;
+        }
+
+        return mapMetadata.Property.Capacity == 0;
+    }
+
+    public static bool MapIsTutorial(int mapId)
+    {
+        MapMetadata mapMetadata = GetMetadata(mapId);
+        if (mapMetadata is null)
+        {
+            return false;
+        }
+
+        return mapMetadata.Property.IsTutorialMap;
+    }
```

Comment:
IMO this should have been a separate PR, as it doesn't relate to event changes. Up to you if you want to split it out.